Resources/doc/reference/action_list.rst

```diff
@@ -94,8 +91,7 @@ Options
 - ``template`` (o): the template used to render the field
 - ``label`` (o): the name used for the column's title
 - ``link_parameters`` (o): add link parameter to the related Admin class when the ``Admin::generateUrl`` is called
-- ``code`` (o): the method name to retrieve the related value
-- ``associated_tostring`` (o): (deprecated, use associated_property option) the method to retrieve the "string" representation of the collection element.
+- ``code`` (o): the method name to retrieve the related value (for example, if you have an `array` type field, you would like to show it in user friendly way, not `[0] => 'Value'`, but in your optional string constructed with method, specified in this option; useful when simple getter is not enough). Notice: works with string-like types (string, text, html)
```

Comment:
"user friendly way, not" => "a user friendly way, and not" . I do not understand the end of this sentence, can you please rephrase it?